### ISSUE TITLE
L1/L2 Keep modules appear to delete TrayInfo (Trac #2146)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#2138]
Recently when investigating possible bugs in custom simulation I was examining the recorded I3TrayInfo objects, but hit a wall at L1/L2, as the `I` frames from previous levels existed but were empty. This looks like the work of the `Keep` module, so I propose that "I3TrayInfo" should be added to the whitelists used in the processing scripts to prevent this kind of information loss. 

Migrated from https://code.icecube.wisc.edu/ticket/2146
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:15:23", 
    "description": "Recently when investigating possible bugs in custom simulation I was examining the recorded I3TrayInfo objects, but hit a wall at L1/L2, as the `I` frames from previous levels existed but were empty. This looks like the work of the `Keep` module, so I propose that \"I3TrayInfo\" should be added to the whitelists used in the processing scripts to prevent this kind of information loss. ", 
    "reporter": "cweaver", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067323910946", 
    "component": "combo reconstruction", 
    "summary": "L1/L2 Keep modules appear to delete TrayInfo", 
    "priority": "minor", 
    "keywords": "", 
    "time": "2018-04-19T15:35:17", 
    "milestone": "", 
    "owner": "mjl5147", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20180922T09:54:49: 
* olivas changed status from "new" to "assigned"
* olivas changed owner from "" to "olivas"
* olivas changed milestone from "" to "Winter Solstice 2018"

**Comment by icecube-trac** [comment]
Trac update at 20181114T14:39:22: olivas changed owner from "olivas" to ""

**Comment by icecube-trac** [comment]
Trac update at 20181204T16:56:52: 
* mjl5147 changed status from "assigned" to "accepted"
* mjl5147 changed owner from "" to "mjl5147"

**Comment by icecube-trac** [comment]
Trac update at 20181207T20:47:55: 
* mjl5147 changed status from "accepted" to "closed"
* mjl5147 commented:

> In changeset 3121:
> 
> ```text
> #!CommitTicketReference repository="IceTray" revision="3121"
> Update the Keep module with a Streams parameter that works identically to the one used in python functions. By default, this will limit Keep's reach to only P and Q frames, saving GCD as well as the TrayInfo and Simulation frame information. Closes #2138
> ```


* mjl5147 changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190111T21:05:51: nwhitehorn commented:

> In changeset 168867:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="168867"
> Update the Keep module with a Streams parameter that works identically to the one used in python functions. By default, this will limit Keep's reach to only P and Q frames, saving GCD as well as the TrayInfo and Simulation frame information. Closes #2138
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:15:23: 
* olivas changed milestone from "Winter Solstice 2018" to ""
* olivas commented:

> Milestone Winter Solstice 2018 deleted
